File: app/src/main/java/com/monk/rxjava2/imetate2/Emitter.java
package com.monk.rxjava2.imetate2;

/**
 * 发射数据
 * @author monk
 * @date 2019-03-01
 */
public interface Emitter<T> {
    void onNext(T t);

    void onCompleted();

    void onError(Throwable throwable);
}

File: module_fragment/src/main/java/com/monk/modulefragment/rxjava2/imetate2/Emitter.kt
package com.monk.modulefragment.rxjava2.imetate2

/**
 * 发射数据
 * @author monk
 * @date 2019-03-01
 */
interface Emitter<T> {
    fun onNext(t: T)
    fun onCompleted()
    fun onError(throwable: Throwable?)
}